aes-gcm/src/lib.rs

```diff
@@ -20,7 +20,7 @@
 //!     Aes256Gcm, Nonce // Or `Aes128Gcm`
 //! };
 //!
-//! let key = Aes256Gcm::generate_key(&mut OsRng);
+//! let key = Aes256Gcm::generate_key(&mut OsRng); // Or let key = Key::<Aes256Gcm>::from_slice(b"an example very very secret key.");
```

Comment:
Some issues wit hthis:

1. Having it on the same (very long) line like this will make it hard to see
2. The example should use a non-panicking conversion
3. Having it in a comment means it won't get doctested